projects/laji/src/app/shared-modules/observation-map/observation-map/observation-map.component.ts

```diff
@@ -554,13 +572,20 @@ export class ObservationMapComponent implements OnChanges, OnDestroy {
 
     if (this.query.coordinates) {
       this.drawDataSubscription?.unsubscribe();
-      this.drawDataSubscription = this.getFeatureCollection().subscribe(featureCollection => {
+      this.drawDataSubscription = this.getDrawFeatureCollection().subscribe(featureCollection => {
         this.drawData = {...this.drawData, featureCollection};
         mapData.push(this.drawData);
         this.mapData = mapData;
       });
     }
   }
+
+  private updateTotalCount() {
+    this.getCountForQuery$(this.query).subscribe(count => {
```

Comment:
Requires unsubscription? I wonder actually, if the query takes a while and the component is destroyed before it completes, does this produce a `this === undefined` error or not (since `this` might not be actually gone/garbage collected yet)

Reply:
Interesting question... I don't think I've ran into `this` being `undefined` on an angular component ever - I'm not sure if the RXJS subscription would somehow keep the instance alive? I added an unsubscription anyways, so this isn't even a theoretical problem anymore :)